puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_kafo_parsers_master.yml

```diff
@@ -0,0 +1,44 @@
+- job:
+    name: test_kafo_parsers_master_temp
+    description: |
+      Tests the Kafo Parsers "master" branch under each Ruby and Puppet configuration.
+    project-type: matrix
+    properties:
+      - github:
+          url: https://github.com/theforeman/kafo_parsers
+    scm:
+      - git:
+          url: https://github.com/theforeman/kafo_parsers
+          per-build-tag: true
+          wipe-workspace: true
+          branches:
+            - master
+    triggers:
+      - github
+      - scm_fifteen_minutes
+    axes:
+      - axis:
+          type: user-defined
+          name: ruby
+          values:
+            - 2.0.0
+            - 2.1
+            - 2.2
+            - 2.3
+      - axis:
+          type: user-defined
+          name: puppet
+          values:
+            - 4.0.0
```

Comment:
This can be reduced to puppet `4.10.0`. We should get puppet 5 in as well but there are some other changes I should get in so it builds.